pages/content/posts/branding.md

```diff
@@ -0,0 +1,34 @@
+
+While Carle is usually smiling, she has five other facial reactions, enabling using slightly modified logos with different moods for different purposes. Her main color is a fantastic purple orchid, #B448FF.
+
+<img width="400" alt="Carle the bookworm with six different facial expressions" src="https://user-images.githubusercontent.com/69888/192943190-9b938307-4084-49e5-86c6-0b270e09f6ad.png">
+
+Using the new logo, the [OpenCollective OWD site](https://opencollective.com/open-web-docs), and the purple orchid color as a starting point, Lola built OWD a new site in [11ty](https://www.11ty.dev/), creating a site with both light and dark themes. Lola claims not to be a designer. We beg to differ. Her two themes are fantastic. The light theme has a lot of white space, is simple and friendly, and is easy to navigate. The dark theme is too. But, unlike many dark site versions, the colors aren’t inverted. Rather, the dark theme uses several gorgeous shades of purple.
+
+After a year of discussing a rebrand, we are so thankful to Lola and Vladimir for their hard work and dedication. We are delighted to officially announce the launch. The new site is available at the same URL as before: [OpenWebDocs.org](https://OpenWebDocs.org). Like everything OWD, the site is [open source and available on Github](https://github.com/openwebdocs/owd-website/). 
+
+The new website provides our visitors with clear, easy way to [learn about Open Web Docs](https://openwebdocs.org/), who we are, what we do, and who funds us. You can learn about what we’ve accomplished and how you can [get involved](https://openwebdocs.org/get-involved/), including [sponsoring us and becoming a member](https://openwebdocs.org/membership/). We have a page about our [open source maintainers and steering and governing committees](https://openwebdocs.org/team/) and a page highlighting our [sponsors and funders](https://openwebdocs.org/sponsors/). The new website also includes [OWD's accomplishments for the past year](https://openwebdocs.org/content/reports/2021/) and a [blog with more details about many of our projects](https://openwebdocs.org/blog/). 
+
+We will continue to update our content with helpful information, articles, announcements, and project successes in the [blog](https://openwebdocs.org/content/posts/) section.
+
+We would like to Lola and Vladimir who gave their time, energy, and talents to make this site and our branding what it is.
+
+For any questions, suggestions, feedback, or comments, please contact us on [Twitter](https://twitter.com/OpenWebDocs) or via the [Open Collective contact form](https://opencollective.com/open-web-docs/contact).
```

Comment:
Or maybe to file an issue on https://github.com/openwebdocs/owd-website/ for feedback?

Reply:
updated with four contact methods. Please review.